donkeycar/parts/tensorrt.py

```diff
@@ -31,12 +31,8 @@ def load(self, model_path):
         uff_model = Path(model_path)
         metadata_path = Path('%s/%s.metadata' % (uff_model.parent.as_posix(), uff_model.stem))
         with open(metadata_path.as_posix(), 'r') as metadata, trt.Builder(self.logger) as builder, builder.create_network() as network, trt.UffParser() as parser:
-            
-            # Without this max_workspace_size setting, I was getting:
-            # Building CUDA Engine
-            # [TensorRT] ERROR: Internal error: could not find any implementation for node 2-layer MLP, try increasing the workspace size with IBuilder::setMaxWorkspaceSize()
-            # [TensorRT] ERROR: ../builder/tacticOptimizer.cpp (1230) - OutOfMemory Error in computeCosts: 0
-            builder.max_workspace_size = 1 << 20 #common.GiB(1)
+
```

Comment:
Is the deleted comment no longer valid?

Reply:
It was never _technically_ correct. Defining the work space size just had the intended side effect. This is also not a very useful comment.